articles/aks/networking-overview.md

```diff
@@ -34,7 +34,7 @@ Nodes in an AKS cluster configured for Advanced networking use the [Azure Contai
 Advanced networking provides the following benefits:
 
 * Deploy your AKS cluster into an existing VNet, or create a new VNet and subnet for your cluster.
-* Every pod in the cluster is assigned an IP address in the VNet, and can directly communicate with other pods in the cluster, and other nodes in the VNet.
+* Every pod in the cluster is assigned an IP address in the VNet, and can directly communicate with other pods in the cluster, and other nodes in the VNet. That means it can use the default service type (clusterip) for pod to pod communication whihc doesn't work in basic. 
```

Comment:
Hi @paulgmiller, can you please clarify the scenario in which you're seeing failed communication, if any? Basic vs. Advanced shouldn't differ in pod-to-pod communication, this sentence is merely clarifying intra-VNet communication between pods, and between pods and other resources in the VNet.

As a side note, "which" is misspelled, and "clusterip" should be "clusterIP" if you're referencing its property name, or "cluster IP" for the concept. Pod-to-pod should also be hyphenated.

Reply:
Not currently seeing failed commication. Inherited an acs-engine cluster we're trying to upgrade to aks and I'm trying to figure out why we're using the azure network plugin rather the default anda a coworked had indicated that with the default kubenet pod to pod communication was bad (not clear if that meant broken or slow).

Reply:
@paulgmiller Hi Paul, I'm going to request this PR be closed in light of the thread currently open in email. Also note that because you're Microsoft FTE, future contributions should be done in the [azure-docs-pr](https://github.com/MicrosoftDocs/azure-docs-pr) private repository. If you need assistance in getting set up, hit me up email. Thanks! -MM